### PR TITLE
Added toggleTouchIdEnrollment to wd client

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

**Comment by dpgraham** [comment]
@imurchie

File: lib/commands.js [hunk 1]
```diff
@@ -3113,4 +3113,19 @@ commands.touchId = function(){
   });
 };
 
+/**
+ * touchId(cb) -> cb(err, touchId)
+ * 
+ * @jsonWire GET /session/:sessionId/appium/simulator/touch_id
```
Comment (imurchie) [inline lib/commands.js]: Is this a `GET` or a `POST` request? It says the former here but the actual call is the latter, which seems more reasonable.

Comment (dpgraham) [inline lib/commands.js]: It's a `POST`. I've fixed the typo for that and `toggle_touch_enroll_id`

File: test/specs/mjson-specs.js [hunk 2]
```diff
@@ -406,6 +406,20 @@ describe("mjson tests", function() {
           .nodeify(done);
       });
 
+      it.only("toggleTouchIdEnrollment", function(done){
```
Comment (imurchie) [inline test/specs/mjson-specs.js]: Probably don't want to have this be exclusive.

**Comment by dpgraham** [comment]
Change made.

**Comment by dpgraham** [comment]
@sebv @admc Is one of you able to review this?

**Pull request merged**